gems/sorbet-runtime/test/types/builder_syntax.rb

```diff
@@ -346,7 +348,13 @@ def self.foo(x); end
           end
 
           it 'setting the default checked level raises if set too late' do
-            # Since the test themselves have evalued sigs, it's too late.
+            Module.new do
+              extend T::Sig
+              sig {void}
+              def self.foo; end
+              foo
+            end
+
```

Comment:
I'm not sure why this one failed. I think it might have something to do with `it` blocks being re-ordered and executed in parallel / different processes? Regardless, I figured it was good hygiene to force one sig explicitly within the `it` block to make sure that setting the default checked level will always have seen at least one sig get forced, independent of the rest of the test suite.